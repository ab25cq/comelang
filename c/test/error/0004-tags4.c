/*
PATTERN: incomplete
PATTERN: c:12:
*/

int
main()
{
	{
		struct X {};
	}
	struct X x;
	return 0;
}
