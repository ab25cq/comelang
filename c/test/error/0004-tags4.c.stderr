cannot use incomplete type in this context at test/error/0004-tags4.c:12:5
    struct X x;
    ^
